src/liquid/components/ld-tooltip/ld-tooltip.tsx

```diff
@@ -158,15 +159,34 @@ export class LdTooltip {
   }
 
   private syncContent = () => {
+    this.isObserverEnabled = false
+
+    // Grab the new content in the slot.
     const tooltipContent = this.contentRef.querySelector('slot').assignedNodes()
 
+    // Delete old content in popper element. Using textContent
+    // is faster than innerHTML as no HTML parsers needs to be
+    // invoked. Instead, this immediately replaces all children
+    // of the tooltip ref with a single #text node.
+    this.tooltipRef.textContent = ''
+
+    // Move original nodes to popper element,
+    // including all event listeners!
     tooltipContent.forEach((node) => {
       copySlottedNodes(node)
-      // We put original node will be put in the tooltipRef, because we cannot
-      // clone event listeners. The original event listeners must be
-      // present on the node that is located in the tooltipRef element.
       this.tooltipRef.appendChild(node)
     })
+
+    // The timeout is required. Without the setTimeout,
+    // there is a possibility that the observer could be
+    // re-enabled immediately after the content synchronization
+    // code has been executed, but before the DOM modifications
+    // have been fully processed. This could result in the
+    // observer detecting incomplete or inconsistent changes,
+    // leading to unexpected behavior.
+    setTimeout(() => {
+      this.isObserverEnabled = true
+    })
```

Comment:
Maybe we can get rid of the timeout by moving this to the `handleSlotChange` method like so:

```ts
if (!this.isObserverEnabled) {
  this.isObserverEnabled = true
  return
}
```

The only thing we would have to check first is, if `handleSlotChange` gets triggered multiple times during the above `forEach` loop (line 175). If not, this change should do the trick, right? I'll pre-approve this PR anyway.

Reply:
That doesn't work unfortunately.